### ISSUE TITLE
compiling kvlt with clojure 1.9 and aleph 0.4.2-alpha8

### ISSUE THREAD

**Issue by konstan** [nervous-systems/kvlt#15]
We are having an issue compiling our code that depends on kvlt with clojure 1.9 (1.9.0-alpha14).  Our code uses `aleph` and after updating the dependency from `[aleph "0.4.1"]` to `[aleph "0.4.2-alpha8"]` we are getting

```
Compiling 1/1 sixsq.slipstream.runproxy.main...
clojure.lang.Compiler$CompilerException: java.lang.RuntimeException: No such var: aleph.http.client-middleware/decorate-url, compiling:(kvlt/platform/http.clj:24:12)
             java.lang.RuntimeException: No such var: aleph.http.client-middleware/decorate-url
```

**Comment by moea** [comment]
@konstan It looks like that var is called `wrap-url` now.  Any chance you could submit a PR which bumps the aleph version and changes `decorate-url` to `wrap-url`?

**Comment by konstan** [comment]
Great.  I'll create the PR.

**Comment by konstan** [comment]
Just for the sake of completeness. 
PR https://github.com/nervous-systems/kvlt/pull/16 with released 0.1.4-SNAPSHOT fixed the issue.  @moea Thanks a lot for the quick response!
